_pages/speaking.md

```diff
@@ -1,104 +1,59 @@
 ---
-description: Information about submitting a proposal to speak at DjangoCon US
-heading: Speaking at DjangoCon US
+description: Information about submitting a proposal to speak at DjangoCon Europe
+heading: Speaking at DjangoCon Europe
 layout: default
 permalink: /speaking/
-published: false
-title: Speaking at DjangoCon US
+title: Speaking at DjangoCon Europe
 ---
 
-{% comment %}
-Our Call for Proposals (CFP) will open soon!
-{% endcomment %}
-
-{% comment %}
-Our Call for Proposals is now closed.
-Decision notifications will be sent by July 8th, 2022.
-{% endcomment %}
-
-Our Call for Proposals is now closed and all decision notifications have been sent.
+We'll be open the Call For Proposals in early January.
+Keep an eye on our [Twitter](https://twitter.com/djangoconeurope) account for notifications!    
 
 {% comment %}
-Our <a href="{{ site.cfp_application }}">Call for Proposals (CFP)</a> is now open!
-Submit your talk or tutorial proposal by the end of day June 10th, 2022 [AoE](https://time.is/compare/0000_10_June_2022_in_Anywhere_on_Earth), and encourage your friends and colleagues to do the same.
-
-Need help with your proposal? We've got mentors and helpful tips on our [Speaker Resources](/speaking/speaker-resources/) page!
-
-<a href="{{site.cfp_application}}" class="button">Submit a Proposal</a>
+**The deadline is Feb 3rd [AoE](https://time.is/compare/0000_3_March_2023_in_Anywhere_on_Earth) 🔥** That is Feb 4th, 2023, 12:00 UTC. You can edit your submissions up until this time.
```

Comment:
The text says Feb 3rd but IIUC the link is for March 3rd

Reply:
In the news posts, I've made it end of 26th Feb anywhere on earth (as we discussed, 26th Feb makes the closing date a Sunday, and then we can assume we might extend for a week to 5th March).
https://github.com/djangocon/2023.djangocon.eu/pull/18/files#diff-8640e7df84f1f4b2c31ade71b9dcd70516f787cfc8768e01c42abb2fd41b5d27R12
I quite like the idea of allowing submissions as long as it's still 26th Feb somewhere, but we can fix a different time - just as long as we make it the same in all the places :slightly_smiling_face: